app/templates/components/user-profil-link.hbs

```diff
@@ -0,0 +1 @@
+<img src={{user.avatarUrl16}} />{{user.login}}
```

Comment:
This component should be named `user-profile-link`

Reply:
Done !